src/csharp/unitypackage/README.md

```diff
@@ -0,0 +1,9 @@
+# Scripts for building gRPC C# package for Unity
+
+Scripts in this directory are of no interest for end users. They are part
+of internal tooling to automate building of the gRPC C# package for Unity.
+
+- `unitypackage_skeleton` - preconfigured .meta files for the unity package
+  layout. The actual assemblies and native libraries will be added into
+  this hierarchy while building the package.
+  
```

Comment:
In general, how were all of the `.meta` files generated? Add notes on that into this README?

Reply:
I will add a note.

Reply:
Done.